docs/pt/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,246 @@
+# Parâmetros do caminho da Url
```

Comment:
Are we using "caminho" instead of "rota"?

Reply:
yes, we are. 
As a programmer I prefer to say "Path" instead of caminho, but as a newbie, I'd prefer to read on docs the word "caminho" as a translate of path, not "rota".

Reply:
That's not what I meant.

We make choices about ambiguous or multiple possible terms.

It looks like we've chosen "rota" in the past, as you can see here: https://fastapi.tiangolo.com/pt/

Reply:
Hi Kludex, sorry about that. BTW I've changed all the words from "caminho" to "rota" accordingly.

As this is my first PR ever, I'm a little confused, should I make a new pr?

Reply:
> As this is my first PR ever, I'm a little confused, should I make a new pr?

For what? There's no need to open a new PR.

Reply:
>  BTW I've changed all the words from "caminho" to "rota" accordingly.

Cool!